README.md

```diff
@@ -71,14 +71,17 @@ be the latest.
 This list presents the known working version combos between the target operating system
 and Erlang/OTP.
 
-| Operating system | Erlang/OTP   | OTP Architecture | Status
-|-                 |-             | -                |-
-| `ubuntu-18.04`   | 17.0 - 25.3  | x86_64, arm64    | ✅
-| `ubuntu-20.04`   | 21.0 - 27    | x86_64, arm64    | ✅
-| `ubuntu-22.04`   | 24.2 - 27    | x86_64, arm64    | ✅
-| `ubuntu-24.04`   | 24.3 - 27    | x86_64, arm64    | ✅
-| `windows-2019`   | 21\* - 25    | x86_64, x86      | ✅
-| `windows-2022`   | 21\* - 27    | x86_64, x86      | ✅
+| Operating system | Erlang/OTP   | OTP Architecture | Status | Supported by GHA?
```

Comment:
Added a column to make it explicit that GitHub no longer supports (thought it might work). We might want to remove this support from the action, too, in the future, but I'd do it in a separate task.